sklearn/cluster/tests/test_hierarchical.py

```diff
@@ -314,6 +314,31 @@ def test_scikit_vs_scipy():
         _hc_cut(n_leaves + 1, children, n_leaves)
 
 
+# Make sure our custom mst_linkage_core gives the same results as scipy's builtin
+@pytest.mark.parametrize('seed', range(5))
+def test_vector_scikit_single_vs_scipy_single():
```

Comment:
you'll need to define the seed as an argument now:

```suggestion
def test_vector_scikit_single_vs_scipy_single(seed):
```

Also there are pep8 issues which will prevent most of the CIs from running, you can check locally by running `make flake8-diff`